### PR TITLE
Updating user list to 0.5. Grey on hover. Adding i18n text on the delete link.

### PR DESCRIPTION
**Pull request by PierrePaul**
Changing delete -> remove
Adding text for screen readers on the Delete link.
Adding hover on <tr> to help select the right row.
Changing the STAFF grey to blue, to fit with the new hover.

https://trello.com/c/99Nq5uDP/208-account-management-content-changes

Screenshot_2020-08-04 Manage accounts — COVID Alert Portal

### CODE REVIEW

File: profiles/templates/profiles/healthcareuser_list.html [hunk 1]
```diff
@@ -24,14 +24,15 @@ <h1>{% trans "Manage accounts" %}</h1>
...
+            <strong class="tag tag--{{ healthcareuser.is_admin|yesno:'green,blue' }}">
               {{ healthcareuser.is_admin|yesno:_("Admin,Staff") }}
             </strong>
           </td>
           <td>
           {% if not healthcareuser.is_superuser and healthcareuser.id != user.id %}
             <a href="{% url 'user_delete' healthcareuser.id %}" class="danger">
-              {% trans "Remove" %}
+              <span aria-hidden="true">{% trans "Delete" %}</span>
+              <span class="visually-hidden">{% blocktrans with email=healthcareuser.email %}Delete account for {email}{% endblocktrans %}</span>
```
Comment (pcraig3) [inline profiles/templates/profiles/healthcareuser_list.html:35]: haha, I think you missed a brace here. Voiceover reads it as the literal string `{email}`.

image

Comment (pcraig3) [inline profiles/templates/profiles/healthcareuser_list.html:35]: I've also done it this way in the past

```html
<span>Delete<span class="visually-hidden"> account for paul@paul.ca</span></span>
```

But your way works 100%.

Comment (PierrePaul) [reply profiles/templates/profiles/healthcareuser_list.html:35]: Yeah, but I find it easier to translate it like this.

**Comment by PierrePaul** [comment]
I updated the invitation table and fixed the missing {} around {email}.

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -599,15 +606,101 @@ msgstr "Droit d’accès"
...
 msgstr "Aucune utilisateur pour le moment"
 
 #: profiles/templates/profiles/privacy.html:12
+#| msgid ""
+#| "\n"
+#| "        <p>The Government of Canada is committed to protect the privacy "
+#| "of its visitors on its websites. This privacy notice summarizes the "
+#| "privacy policy and practices of Canadian Digital Service websites, which "
+#| "are under the responsibility of the <a href=\"https://www.canada.ca/en/"
+#| "treasury-board-secretariat.html\">Treasury Board of Canada Secretariat</"
```
Comment (pcraig3) [inline locale/fr/LC_MESSAGES/django.po:625]: Did something here change?

File: profiles/static/scss/components/_table.scss [hunk 3]
```diff
@@ -27,6 +27,9 @@ main table {
       padding-right: 0;
     }
   }
+  tbody tr:hover {
+    background-color: $color-grey-light;
+  }
```
Comment (pcraig3) [inline profiles/static/scss/components/_table.scss:32]: Let's remove this.

File: locale/fr/LC_MESSAGES/django.po (hunk 2 above)
Comment (PierrePaul) [reply locale/fr/LC_MESSAGES/django.po:626]: Not the content, but I think there was a merge that didnt go well with main at some point. So I had to take the french content manually and add it back to django.po.

**Pull request merged**